### DISCUSSION TITLE
Empty exception message

### DISCUSSION THREAD

**Discussion by AlexVanK** [directus/directus#15650] [Archive]
Hello,

This used to work:
![image](https://user-images.githubusercontent.com/56647386/191070538-30def1ea-3a2e-4360-83b1-6639d8c27de6.png)

but now it only throws an empty box:
![image](https://user-images.githubusercontent.com/56647386/191069951-65bb3c73-870b-4fe8-9c67-0e31f40c3f50.png)

However the error is in fact there:
![image](https://user-images.githubusercontent.com/56647386/191070069-33c23113-4890-44ae-b166-ab80a930e07d.png)

--

Did I miss something in the recent updates? How do I get this to work again?

**Comment by azrikahar** [comment]
This seems to be related to the fix in #15520 (and #15579), but I was not able to reproduce the empty dialog. Here's the test I did locally:

1. Create a Filter hook for `test` collection's `create` event:

    ```js
    module.exports = ({ filter }, { exceptions }) => {
	    const { InvalidPayloadException } = exceptions;
    
	    filter('test.items.create', () => {
		    throw new InvalidPayloadException('This is an invalid payload from a filter hook');
	    });
    };
    ```

2. Create an item in `test` collection
3. Receive `This is an invalid payload from a filter hook` error message which is the same from the thrown InvalidPayloadException's message.

    https://user-images.githubusercontent.com/42867097/191186467-a965415f-be6f-4c4e-a892-8a61e76b19ca.mp4

Are you triggering this somewhere that is not a filter hook event? Or are you using the latest production version `9.18.0`, or a fork?
